buildout.cfg

```diff
@@ -1,28 +0,0 @@
-[buildout]
-parts =
-    doc
-    test
-develop = .
-allow-picked-versions = true
-show-picked-versions = true
-
-[versions]
-configparser = < 5
-docutils = >= 0.15.2
-pytest = >=4.6, < 5
-zipp = < 2
-importlib-metadata = < 3
-
-[doc]
-recipe = zc.recipe.egg
-eggs = gocept.package [doc]
-       gocept.recordserialize
-scripts = doc
```

Comment:
There is currently no other way to create the documentation, so we should keep this file.

Reply:
There is actually not that much documentation in this package, but okay.